.circleci/config.yml

```diff
@@ -44,11 +54,41 @@ jobs:
       - save_cache:
           key: go-modules-v1-{{ checksum "go.sum" }}
           paths:
-            - /go/pkg/mod
-
+            - /$GOPATH/pkg/mod
+  publish_release:
+    executor: golang
+    environment:
+      GHR_VERSION: v0.13.0
+    steps:
+      - run:
+          name: Install ghr
+          command: GO111MODULE="on" go get -v github.com/tcnksm/ghr@${GHR_VERSION}
+      - attach_workspace:
+          at: ./artifacts
+      - run:
+          name: Publish GitHub Release
+          command: |
+            ghr \
+              -token ${GITHUB_TOKEN} \
+              -owner ${CIRCLE_PROJECT_USERNAME} \
+              -repository ${CIRCLE_PROJECT_REPONAME} \
+              -commitish ${CIRCLE_SHA1} \
+              $(./artifacts/forge -version) \
+              artifacts/
 workflows:
   version: 2
   main:
     jobs:
       - build:
           context: org-global
```

Comment:
Missed this before, but what is `org-global`?

Reply:
It's a context that I created inside circleci for the dominodatalab org. It contains docker registry credentials that we can re-use.

https://github.com/dominodatalab/forge/blob/196797214c491e740794269f4bdf961a4d83bc60/.circleci/config.yml#L52)